docs/how-to-guides/how-to-setup-triggers.md

```diff
@@ -0,0 +1,101 @@
+---
+id: how-to-setup-triggers
+title: How to setup triggers
+sidebar_label: Triggers
+---
+
+Memgraph supports **database triggers** that can be executed if a particular type of event occurs. Events are related to changes in data, such as created, updated, and deleted nodes or relationships. 
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/triggers.md) 
+
+
+## How to create a trigger?
+
+You can create trigger by executing cypher commands. Creating a trigger will ensure that some procedural code is executed on certain type of event. 
+All triggers you will create are persisted on the disk, this means triggers will be active on database reruns and no information is lost.
+
+### Trigger execution upon node creation
+
+Node creation can be most common event your Memgraph database can react to. For example, you need to updated some value on created node. 
+If you need a trigger after the node(vertex) has been created, you can set up the following trigger: 
+
+```cypher
+CREATE TRIGGER createVertex
+ON () CREATE AFTER COMMIT EXECUTE
+UNWIND createdVertices AS createdVertex
+SET createdVertex.created = timestamp()
+```
+
+Here the trigger's name is `createdVertex` and should be unique. Cypher command  `ON CREATE` defines on what event will trigger be executed. `AFTER COMMIT EXECUTE` means the trigger will be executed after changes have been committed to the database. For the sake of ease of use, triggers have a set o **predefined variables**. One is `createdVertices`, a list of all created nodes (vertices). Here the list is being unwinding by cypher command `UNWIND` 
+To find a complete list of predefined variables, supported operations, and configuration details, look at triggers [reference guide](https://memgraph.com/docs/memgraph/reference-guide/triggers).
+
+In this trigger via `SET createdVertex.created = timestamp()`  command, new node is getting a timestamp after creation.
+
+### Trigger execution upon node update
+
+Node property updates are common in graphs,in order to react to them you can create trigger for that type of event: 
+
+```cypher
+CREATE TRIGGER updateVertex
+ON () UPDATE AFTER COMMIT EXECUTE
+UNWIND updatedVertices AS updatedVertex
+SET updatedVertex.updated_at = timestamp()
+
+```
+The trigger for node updates is almost the same as node creation. Notice different predefined variable `updatedVertices` and `ON UPDATE` cypher command.
```

Comment:
... is almost the same as the trigger for node creation.?

Cypher is always written with capital C, and Cypher doesn't have commands but clauses and sub-clauses... at least that's how we refer to them... so I'm not gonna fix that anymore, you can go through the text to find if there are more occurrences where commands should be actually clauses, and cypher should be Cypher